lib/icingadb/icingadb.cpp

```diff
@@ -56,7 +56,6 @@ void IcingaDB::Start(bool runtimeCreated)
 	m_Rcon->SetConnectedCallback([this](boost::asio::yield_context& yc) {
 		m_WorkQueue.Enqueue([this]() { OnConnectedHandler(); });
 	});
```

Comment:
I'd also move this together with the call to `m_Rcon->Start()` and add a comment stating that this callback depends on the child connections. Moving when the callback is set makes no functional difference, but I think it would be nice to have all this grouped together (together with a comment why this is done somewhere else than where the object is created).